distribution/pom.xml

```diff
@@ -174,10 +234,82 @@
                             <descriptors>
                                 <descriptor>src/main/assembly/bin.xml</descriptor>
                             </descriptors>
-                            <finalName>wso2am-micro-gw-${pom.version}</finalName>
+                            <finalName>wso2am-micro-gw-toolkit-${pom.version}</finalName>
+                            <appendAssemblyId>false</appendAssemblyId>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>micro-gateway-runtime-linux</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>attached</goal>
+                        </goals>
+                        <configuration>
+                            <filters>
+                                <filter>${basedir}/src/main/assembly/filter.properties</filter>
+                            </filters>
+                            <descriptors>
+                                <descriptor>src/main/assembly/gw_runtimes/gw_runtime_linux.xml</descriptor>
+                            </descriptors>
+                            <finalName>wso2am-micro-gw-linux-${pom.version}</finalName>
```

Comment:
Shouldn't we add OS arch for the distribution name? AFAIK we only support x64 as we discussed. So shouldn't it be there in the file name? like `wso2am-micro-gw-linux-x86_64-${pom.version}`